docs/articles/documentation/templates/boundtestrun.md

```diff
@@ -0,0 +1,9 @@
+### options.boundTestRun
+
+**Type**: Object
+
+You can use the `boundTestRun` option to call a client function from a Node.js callback.
```

Comment:
Change to:
Use the `boundTestRun` option to call a client function from a Node.js callback.